charts/studio/README.md

```diff
@@ -94,6 +95,10 @@ A Helm chart for Kubernetes
 | postgresql.fullnameOverride | string | `"studio-postgresql"` | Postgres name override |
 | postgresql.global.postgresql.auth.database | string | `"iterativeai"` | Postgres database |
 | postgresql.global.postgresql.auth.postgresPassword | string | `"postgres"` | Postgres password |
+| ray | object | `{"enabled":false,"head":{"resources":{"cpu":"500m","memory":"512M"}},"image":"rayproject/ray:2.4.0-py310-cpu","version":"2.4.0","worker":{"replicas":{"maximum":4,"minimum":1},"resources":{"cpu":"500m","memory":"512M"}}}` | Ray cluster settings group used by DVCx |
```

Comment:
Let's either delete this line or clean it up. It seems like helm-docs just generated a JSON structure of the YAML. It makes it really confusing :confused:

Reply:
It also happens for many other items in the `README.md` table. I guess fixing that is out of scope for this pull request. 🤔

Reply:
Fair enough! I didn't notice. Let's follow up later